pyproject.toml

```diff
@@ -38,16 +38,17 @@ classifiers = [
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3 :: Only",
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.12"
+python = ">=3.8,<3.12"
 fastapi = ">=0.78,<0.86"
-pydantic = ">=1.10.0,<2.0.0"
+pydantic = ">=1.9.1,<2.0.0"
```

Comment:
Hmm we do need pydantic `>= 1.10.0` for python 3.11. 🤔 Maybe there is a way to declare that?

Reply:
Hmm that's surprising, look at https://github.com/arthurio/fastapi-filter/pull/67